external/googleapis/update_libraries.sh

```diff
@@ -154,18 +154,13 @@ declare -A -r LIBRARIES=(
   ["iam"]="$(
     printf ",%s" \
       "@com_google_googleapis//google/iam/credentials/v1:credentials_cc_grpc" \
-      "@com_google_googleapis//google/iam/admin/v1:admin_cc_grpc" \
-      "@com_google_googleapis//google/iam/v2:iam_cc_grpc"
+      "@com_google_googleapis//google/iam/admin/v1:admin_cc_grpc"
   )"
   # We already have libraries, different from `iam_protos` that compile
   # the protos defined in the following entry. Thus, we cannot merge
   # the following entry into the existing `iam` entry, that would
   # introduce either ODR-violations, or break backwards compatibility.
-  ["iam_policy"]="$(
-    printf ",%s" \
-      "@com_google_googleapis//google/iam/v1:iam_cc_grpc" \
-      "@com_google_googleapis//google/iam/v1/logging:logging_cc_grpc"
```

Comment:
Looks like lost the iam/v1/logging protos?  I am not sure it was a good idea to include them in the past, but since we did, we need to keep them.